File: SiremGy.Models/Users/UserDetailModel.cs
using System.Collections.Generic;
using SiremGy.BLL.Models.Photos;
using System;

namespace SiremGy.BLL.Models.Users
{
    public class UserDetailModel : ModelBase
    {
        public string Nickname { get; set; }
        public string Gender { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string Address { get; set; }
        public string Introduction { get; set; }
        public string LookingFor { get; set; }
        public string Intrests { get; set; }
        public UserModel User { get; set; }
        public PhotoModel Photo { get; set; }
        public IEnumerable<PhotoModel> Photos { get; set; }
        public int Age { get; set; }
        public DateTime Birthday { get; set; }
    }
}

File: SiremGy-SPA/src/app/models/UserDetailModel.ts
import { UserModel } from './userModel';
import { PhotoModel } from './PhotoModel';

export interface UserDetailModel {
    Id: number;
    Nickname: string;
    Gender: string;
    Age: number;
    Country: string;
    City: string;
    Address: string;
    Introduction: string;
    LookingFor: string;
    Intrests: string;
    Birthday?: Date;
    User?: UserModel;
    Photo?: PhotoModel;
    Photos?: PhotoModel[];
}
